onnxruntime/core/framework/parallel_executor.cc

```diff
@@ -95,6 +101,9 @@ void ParallelExecutor::RunNodeAsyncInternal(size_t p_node_index,
   size_t node_index = p_node_index;
   bool keep_running = true;
   auto graph_viewer = session_state.GetGraphViewer();
+  TimePoint sync_time_begin;
+  TimePoint kernel_begin_time;
+  bool f_profiler_enabled = session_state.Profiler().FEnabled();
```

Comment:
bool profiler_enabled = session_state.Profiler().Enabled() reads better?

Reply:
same with the function name, f means flag here.